chatbox_tests/test_arsenic.py

```diff
@@ -169,10 +220,14 @@ async def test_previous_messages(server, user):
         await new_msg_box.clear()
         await new_msg_box.send_keys(message)
         await send_button.click()
+        time.sleep(1)
```

Comment:
It is a minor thing, but it is safer to use "waitFor..." methods instead of sleeps.